src/backtrace/libunwind.rs

```diff
@@ -66,7 +66,11 @@ impl Frame {
         //
         // Note the `skip_inner_frames.rs` test is skipped on macOS due to this
         // clause, and if this is fixed that test in theory can be run on macOS!
-        if cfg!(target_os = "macos") || cfg!(target_os = "ios") {
+        if cfg!(target_os = "macos")
+            || cfg!(target_os = "ios")
+            || cfg!(target_os = "tvos")
+            || cfg!(target_os = "watchos")
+        {
```

Comment:
Can we just use `cfg!(target_vendor = "apple")`?

Reply:
Sounds good to me.